### ISSUE TITLE
check_include: SAP Include not found

### ISSUE THREAD

**Issue by ed-holland** [abaplint/abaplint#2252]
We use the sap include LBTCHDEF in our function group.
![image](https://user-images.githubusercontent.com/53995991/142993505-c13b286c-0adc-4e37-98ba-88cb71da841f.png)

The dependency repository now does not contain it anymore.
Is this related to issue #5134 (abapgit) also? [see issue](https://github.com/abapGit/abapGit/issues/5155#event-5658723157)

This include can not be renamed as it is a SAP original.

I only see two options:
1. The check is not done for the dependency generation?
2. This include becomes part of the exceptions that need to be generated extra

**Comment by mbtools** [comment]
I would put SAP object definitions in a separate repo (like https://github.com/abaplint/deps) and refer to it in your `abaplint.json`. Basically, the same if you reuse other SAP objects like interfaces. 

You will need the content of the include to be parsed by abaplint or the parts of your code that reference for example variables or constants defined in the include will raise issues.

**Comment by ed-holland** [comment]
It is already implemented this way. The issue is that the include is not generated in the dependency repository.
I've added the include to the additional objects (parameter in the deps generator) which works fine.

This issue is caused by the decision to not allow fugr includes to be part of other code (we discussed this in another issue).

As this is working fine (just needed to add 7 objects) i'll close this issue.
